bunsen/extension-structure-definitions/src/main/resources/stu3-us-core-definitions/StructureDefinition-us-core-race.json

```diff
@@ -0,0 +1 @@
nsion:ombcategory","path":"Extension.extension","sliceName":"ombCategory","short":"American Indian or Alaska Native|Asian|Black or African American|Native Hawaiian or Other Pacific Islander|White","definition":"The 5 race category codes according to the [OMB Standards for Maintaining, Collecting, and Presenting Federal Data on Race and Ethnicity, Statistical Policy Directive No. 15, as revised, October 30, 1997](https://www.whitehouse.gov/omb/fedreg_1997standards).","min":0,"max":"5","type":[{"code":"Extension"}],"mustSupport":true,"isModifier":false,"mapping":[{"identity":"iso11179","map":"/ClinicalDocument/recordTarget/patientRole/patient/raceCode"}]},{"id":"Extension:race.extension:ombcategory.url","path":"Extension.extension.url","min":1,"max":"1","type":[{"code":"uri"}],"fixedUri":"ombCategory"},{"id":"Extension:race.extension:ombcategory.valueCoding","path":"Extension.extension.valueCoding","min":1,"max":"1","type":[{"code":"Coding"}],"binding":{"strength":"required","description":"The 5 race category codes according to the [OMB Standards for Maintaining, Collecting, and Presenting Federal Data on Race and Ethnicity, Statistical Policy Directive No. 15, as revised, October 30, 1997](https://www.whitehouse.gov/omb/fedreg_1997standards).","valueSetReference":{"reference":"http://hl7.org/fhir/us/core/ValueSet/omb-race-category"}}},{"id":"Extension:race.extension:detailed","path":"Extension.extension","sliceName":"detailed","short":"Extended race codes","definition":"The 900+ CDC race codes that are grouped under one of the 5 OMB race category codes:.","min":0,"max":"*","type":[{"code":"Extension"}],"isModifier":false,"mapping":[{"identity":"iso11179","map":"/ClinicalDocument/recordTarget/patientRole/patient/sdtc:raceCode"}]},{"id":"Extension:race.extension:detailed.url","path":"Extension.extension.url","min":1,"max":"1","type":[{"code":"uri"}],"fixedUri":"detailed"},{"id":"Extension:race.extension:detailed.valueCoding","path":"Extension.extension.valueCoding","min":1,"max":"1","type":[{"code":"Coding"}],"binding":{"strength":"required","description":"The [900+ CDC Race codes](http://www.cdc.gov/phin/resources/vocabulary/index.html) that are grouped under one of the 5 OMB race category codes.","valueSetReference":{"reference":"http://hl7.org/fhir/us/core/ValueSet/detailed-race"}}},{"id":"Extension:race.extension:text","path":"Extension.extension","sliceName":"text","short":"Race Text","definition":"Plain text representation of the race concept(s).","min":1,"max":"1","type":[{"code":"Extension"}],"mustSupport":true,"isModifier":false},{"id":"Extension:race.extension:text.url","path":"Extension.extension.url","min":1,"max":"1","type":[{"code":"uri"}],"fixedUri":"text"},{"id":"Extension:race.extension:text.valueString","path":"Extension.extension.valueString","min":1,"max":"1","type":[{"code":"string"}]},{"id":"Extension:race.url","path":"Extension.url","min":1,"max":"1","fixedUri":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-race"}]}}
```

Comment:
ditto for this and any other repeated SD files.

Reply:
Done